include/counter.h

```diff
@@ -329,6 +330,7 @@ static inline int counter_cancel_channel_alarm(struct device *dev,
  * @retval -EBUSY if any alarm is active.
```

Comment:
add -ENOTSUP error if option is not supported

Reply:
Updated.